error building: 'TIFFCreateEXIFDirectory' was not declared in this scope

Hi,
I have downloaded oiio-Release-1.5.13
typed 'make' and I get the following errors:

    [ 78%] Building CXX object src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/tiff.imageio/tiffoutput.cpp.o
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp: In member function 'bool OpenImageIO::v1_5::TIFFOutput::write_exif_data()':
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:549:39: error: 'TIFFCreateEXIFDirectory' was not declared in this scope
         if (TIFFCreateEXIFDirectory (m_tif) != 0) {
                                           ^
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:586:5: error: 'uint64' was not declared in this scope
         uint64 dir_offset = 0;
         ^
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:586:12: error: expected ';' before 'dir_offset'
         uint64 dir_offset = 0;
                ^
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:587:45: error: 'dir_offset' was not declared in this scope
         if (! TIFFWriteCustomDirectory (m_tif, &dir_offset)) {
                                                 ^
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:587:55: error: 'TIFFWriteCustomDirectory' was not declared in this scope
         if (! TIFFWriteCustomDirectory (m_tif, &dir_offset)) {
                                                           ^
    /home/.../oiio-Release-1.5.13/src/tiff.imageio/tiffoutput.cpp:594:43: error: 'dir_offset' was not declared in this scope
         TIFFSetField (m_tif, TIFFTAG_EXIFIFD, dir_offset);
                                               ^
    make[3]: *** [src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/tiff.imageio/tiffoutput.cpp.o] Error 1
    make[3]: Leaving directory `/home/.../oiio-Release-1.5.13/build/linux64'
    make[2]: *** [src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/all] Error 2
    make[2]: Leaving directory `/home/.../oiio-Release-1.5.13/build/linux64'
    make[1]: *** [all] Error 2
    make[1]: Leaving directory `/home/.../oiio-Release-1.5.13/build/linux64'
    make: *** [cmake] Error 2

Tried 1.5.12 too, with the same results.

--- a/src/tiff.imageio/tiffoutput.cpp
+++ b/src/tiff.imageio/tiffoutput.cpp
@@ -542,7 +542,7 @@ TIFFOutput::put_parameter (const std::string &name, TypeDesc type,
 bool
 TIFFOutput::write_exif_data ()
 {
-#if TIFFLIB_VERSION >= 20120922
+#if defined(TIFF_VERSION_BIG) && TIFFLIB_VERSION >= 20120922
     // Older versions of libtiff do not support writing Exif directories
 
     // First, see if we have any Exif data at all

